template-parts/fixed-cta-stacked.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Template for Displaying T&C header fixed-cta.
+ *
+ * @package info.nu.edu
+ */
+
+?>
+
+<div class="fixed-cta--stacked d-none">
+	<div class="container">
+		<div class="row">
+			<h3>
+				Don't delay - new classes begin every 30 days!
+			</h3>
+			<a href="#form-offset" class="button button--yellow">
+				Request Info
+			</a>
```

Comment:
- Inline elements should keep their text inline instead of spaced out.
- For the both hardcoded pieces of text, can you add a custom filter so child theme can override it without having to submit a full template part file.